### PR TITLE
Adding MedicationOrder DSTU2 documentation

### PR DESCRIPTION
**Pull request by cerbs**
Adding documentation for MedicationOrder DSTU2

### CODE REVIEW

File: content/dstu2/medication-order.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+`timing-boundsperiod-end`| N |[`date`](http://hl7.org/fhir/DSTU2/search.html#date)| The [stop time](http://hl7.org/fhir/DSTU2/MedicationOrder-definitions.html#MedicationOrder.dosageInstruction.timing) of the order. Must be prefixed by `le` (currently only support querying for orders completed prior to a certain time). Not accepted when querying active or draft orders. Example: `le2014-05-19T20:54:02.000Z`
+[`_count`](http://hl7.org/fhir/DSTU2/search.html#count)| N |[`number`](http://hl7.org/fhir/DSTU2/search.html#number)| The number of items to include in a page. Currently ignored if querying for active or draft statuses. Example: `50`
+
+\* Note: The current implementation requires separating the query for active medication orders from inactive. This is a technical limitation that we hope to remove in the future.
+
+* Active statuses: `active`, `draft`
+* Inactive statuses: `completed`, `on-hold`, `stopped`
+
+Also, the `superseded` and `entered-in-error` statuses are not currently supported (the server will return an error if requested).
```
Comment (dennispatterson) [inline content/dstu2/medication-order.md]: superseded is no longer a status

**Comment by dennispatterson** [comment]
:+1:

**Comment by daliboz** [comment]
+1

**Comment by cerbs** [comment]
Rebasing and squashing

**Pull request merged**